apps/site/src/components/layout/Header.astro

```diff
@@ -0,0 +1,103 @@
+---
+import { Icon } from "astro-icon/components";
+import Logo from "./Logo.astro";
+
+const links = [
+  {
+    href: "/",
+    text: "Home",
+  },
+  {
+    href: "/projects",
+    text: "Projects",
+  },
+  {
+    href: "/docs",
+    text: "Documentation",
+  },
+  {
+    href: "/blog",
+    text: "Articles",
+  },
+];
+
+const socialLinks = [
+  {
+    href: "/",
+    text: "Twitter (X)",
+    icon: "ri:twitter-x-fill",
+  },
+  {
+    href: "/docs",
+    text: "GitHub",
+    icon: "ri:github-fill",
+  },
+  {
+    href: "/blog",
+    text: "Discord",
+    icon: "ri:discord-fill",
+  },
+];
+---
+
+<header
+  class="bg-white sticky top-0 z-20 border-b border-b-gray-200 font-heading"
+>
+  <div
+    class="p-2 max-w-7xl mx-auto flex justify-center relative items-center h-16"
+  >
+    <div class="absolute left-2">
+      <a
+        href="/"
+        class="font-extrabold uppercase text-3xl text-primary-950 flex items-center gap-1"
+      >
+        <Logo />
+        <div>
+          <span class="text-primary-600">Astro</span>licious
+        </div>
+      </a>
+    </div>
+    <div class="hidden lg:flex items-center justify-center">
+      {
+        links.map(({ href, text }) => (
+          <a
+            {href}
+            class="py-1.5 px-3 font-extrabold uppercase text-lg rounded-md hover:bg-primary-200 transition-colors text-primary-950"
+          >
+            {text}
+          </a>
+        ))
+      }
+    </div>
+    <div class="absolute right-2">
+      <div class="flex items-center justify-center gap-1">
+        {
+          socialLinks.map(({ href, text, icon }) => (
+            <a
+              {href}
+              target="_blank"
+              class="p-1 font-extrabold uppercase text-lg rounded-full hover:bg-primary-200 transition-colors text-primary-950"
+            >
+              <Icon name={icon} class="size-5" />
+              <span class="sr-only">{text}</span>
+            </a>
```

Comment:
Maybe we should set `target="_blank"` for the `socialLinks` or if the link is external in general?

Reply:
yeah that would be great, not sure what's the recommended way

Reply:
Not sure how to implement a good generic solution, just set them manually for the `socialLinks` for now.